### ISSUE TITLE
Fix typo in "reccomend_textbook"

### ISSUE THREAD

**Issue by nathanmsmith** [hotseatio/hotseat.io#41]
The `Review` model has a `reccomend_textbook` (sic) column. Let's add a migration to fix that typo!

**Comment by briannarenni** [comment]
Hi, I'm looking for a good first issue to use as a first time contribution! Is this the only file that needs the typo fix, or does the typo appear in other places?

Only asking since I've never worked with Ruby (I stumbled on this from the HTML tag) and this seems like a simple typo issue not needing build.

**Comment by nathanmsmith** [comment]
Hey @briannarenni! We'd want to update every file where this occurs:
```
test/factories.rb
99:    reccomend_textbook { false }

test/system/reviews_system_test.rb
91:    assert review.reccomend_textbook
165:    assert review.reccomend_textbook

test/models/review_test.rb
101:                     reccomend_textbook: false)
121:            label: :reccomend_textbook,

test/helpers/course_helper_test.rb
138:    describe "reccomend_textbook" do
141:          label: :reccomend_textbook,
149:          label: :reccomend_textbook,
157:          label: :reccomend_textbook,

app/controllers/reviews_controller.rb
117:        reccomend_textbook: review_params.textbook,

app/helpers/course_helper.rb
112:    when :reccomend_textbook
150:    when :reccomend_textbook

app/models/review.rb
70:    textbook_reccomendations = reviews.where.not(reccomend_textbook: nil)
71:                                      .group(:reccomend_textbook).count
102:                     label: :reccomend_textbook,
212:        label: :reccomend_textbook,
213:        value: reccomend_textbook,
```

We also need to update the database column, to do that we'll need to use a [Rails migration](https://edgeguides.rubyonrails.org/active_record_migrations.html) using the [`rename_column`](https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column) function. You can see an example [here](https://github.com/hotseatio/hotseat.io/blob/main/db/migrate/20201231005534_rename_instructor_column.rb#L10). Read through the migration guide and let me know if you're still interested! No worries if not too :)

**Comment by briannarenni** [comment]
> Hey @briannarenni! We'd want to update every file where this occurs:
> 
> ```
> test/factories.rb
> 99:    reccomend_textbook { false }
> 
> test/system/reviews_system_test.rb
> 91:    assert review.reccomend_textbook
> 165:    assert review.reccomend_textbook
> 
> test/models/review_test.rb
> 101:                     reccomend_textbook: false)
> 121:            label: :reccomend_textbook,
> 
> test/helpers/course_helper_test.rb
> 138:    describe "reccomend_textbook" do
> 141:          label: :reccomend_textbook,
> 149:          label: :reccomend_textbook,
> 157:          label: :reccomend_textbook,
> 
> app/controllers/reviews_controller.rb
> 117:        reccomend_textbook: review_params.textbook,
> 
> app/helpers/course_helper.rb
> 112:    when :reccomend_textbook
> 150:    when :reccomend_textbook
> 
> app/models/review.rb
> 70:    textbook_reccomendations = reviews.where.not(reccomend_textbook: nil)
> 71:                                      .group(:reccomend_textbook).count
> 102:                     label: :reccomend_textbook,
> 212:        label: :reccomend_textbook,
> 213:        value: reccomend_textbook,
> ```
> 
> We also need to update the database column, to do that we'll need to use a [Rails migration](https://edgeguides.rubyonrails.org/active_record_migrations.html) using the [`rename_column`](https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column) function. You can see an example [here](https://github.com/hotseatio/hotseat.io/blob/main/db/migrate/20201231005534_rename_instructor_column.rb#L10). Read through the migration guide and let me know if you're still interested! No worries if not too :)

Alright, I'm working on something for at most another hour and I should be done. I'll give it a read through and let you know!

**Comment by briannarenni** [comment]
I'm going to fork it and try this out, I think I've got it down just for the column name change. If I create a migration file in my fork without running it, you'll be able to review it first with no affect to the DB right? I'll go ahead and try it out.

**Comment by nathanmsmith** [comment]
Yes, you can just create the migration file! I'm happy to review and test it.

**Comment by briannarenni** [comment]
Oh great timing, I think the only thing I'm not totally sure of is this in your example:
`rename_column(:sections, :instructors, :registrar_instructors)`
Is this change in the same sections table?

**Comment by nathanmsmith** [comment]
Good question! No, it's actually on the `reviews` table!

**Comment by briannarenni** [comment]
Alright, I made the fork so I should have a PR in the next few minutes then! I love trying new things, I just want to make sure I don't nuke a table haha.

**Comment by briannarenni** [comment]
Agh. Unfortunately I have Ruby installed, but am running into an issue with my PATH and the pre-installed older version on my Mac. I'm going to spend some time trying to fix it but for times sake, I won't be able to work on the PR. If I can get it using the correct version tomorrow I'll let you know though if no one else has done this issue.

**Comment by TimothyGu** [comment]
Hi @briannarenni, has there been any progress on this issue?

**Comment by briannarenni** [comment]
> Hi @briannarenni, has there been any progress on this issue?

I couldn't get Ruby to work, I ended up finding issues in my shell and fixing them that night. I meant to comment and release this to someone else but it slipped my mind, apologies for that.

**Comment by subashravichandran** [comment]
Hi, I'd like to work on this issue.

**Comment by Shweta-Tapele** [comment]
Hello, I would like to try this out and would be my first contribution.

**Comment by nathanmsmith** [comment]
sorry, I currently don't have the time to review contributions for outside contributors! Will remove the good first issue tage
